modules/bridge/src/main/ts/ephox/bridge/api/PublicMenu.ts

```diff
@@ -29,5 +30,11 @@ export {
   ContextMenuApi,
   ContextMenuContents,
   ContextMenuItem,
-  ContextSubMenu
+  ContextSubMenu,
+
+  CardMenuItemSpec,
+  CardMenuItem,
```

Comment:
These shouldn't be here, as it's an internal-only spec (likewise with ContainerItem below). This module should only have public types that are used when registering the component by the APIs.

Also, should this have `CardContainerSpec` and all those other relevant types exposed?